lib/tower_cli/resources/credential.py

```diff
@@ -144,6 +144,8 @@ class Resource(models.Resource):
     vault_password = models.Field(password=True, required=False)
 
     def create(self, **kwargs):
+        """Create a credential.
+        """
```

Comment:
This is a good catch. I was vaguely aware that we were missing help text in some places, but I had not got around to making an issue for it.